lib/bolt_server/transport_app.rb

```diff
@@ -54,7 +54,8 @@ def initialize(config)
 
       body = JSON.parse(request.body.read)
       schema_error = JSON::Validator.fully_validate(@schemas["ssh-run_task"], body)
-      return [400, schema_error.join] if schema_error.any?
+      # Do we want to construct an error hash, or just this?
```

Comment:
schema_error is an array. I think we should at least return the array rather than joining it.